File: sangi_practice/sangi_practice_06/model_answers/cpp/main.cpp
#include <iostream>
using namespace std;

int main() {
    int p, d, t;
    cin >> p >> d >> t;
    int discounted = p * (100 - d) / 100;
    int total = discounted * (100 + t) / 100;
    cout << total << "\n";
    return 0;
}

File: sangi_practice/sangi_practice_06/model_answers/java/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int p = sc.nextInt(), d = sc.nextInt(), t = sc.nextInt();
        int discounted = p * (100 - d) / 100;
        int total = discounted * (100 + t) / 100;
        System.out.println(total);
    }
}
